recipes/beyla-golden-signals/gce/README.md

```diff
@@ -0,0 +1,88 @@
+# eBPF Golden Signals with Beyla on GCE
+
+This recipe demonstrates how to configure the Google Cloud Ops Agent (installed on your Google Compute Engine instance) to produce golden signal metrics from [Beyla](https://github.com/grafana/beyla) http
+metrics and send those metrics to [Google Managed Service for
+Prometheus](https://cloud.google.com/stackdriver/docs/managed-prometheus).
+
+In this recipe, Beyla is configured to generate http metrics & traces from an application running on a GCE instance without introducing any code changes. The sample application used for this recipe is a simple Java server-client application located in the [sample-apps/java](../../../sample-apps/java/) directory of this repository.
```

Comment:
we probably shouldn't have traces enabled for this demo -- just the metrics.

Reply:
Is it to keep the scope of the demo limited or to keep the cost of running the demo low for the user?

We can disable trace export from the Ops Agent so that it doesn't show up in the user's console.

Reply:
More the latter (cost).  You are using a demo application, but ideally these deployment instructions would work for a production service as well.  The spans it generates aren't very useful, since context propagation doesn't work. All you can do with them is make metrics :).